### ISSUE TITLE
Allow access to alternative metaverse servers

### ISSUE THREAD

**Issue by mirrorwolf** [vircadia/vircadia#1175]
I tried hosting my own private metaverse server so I wouldn't have to make a centralized account, and it worked great! There's just one problem:

The official client can't connect to it!

Seeing as the domain server already has an environment variable you can set to change this, I thought I'd add the same thing to the client (#1174). It'd be nice to have actual UI to set this from the login screen, though.

**Comment by mirrorwolf** [comment]
So, apparently this environment variable is already a thing, and already works perfectly! The pull request is _completely_ redundant.

Now we just need to build UI for this, and a setting to persist it.

**Comment by JulianGro** [comment]
Yeah the system isn't really fully fledged yet. It is supposed to be federated like matrix.

**Comment by daleglass** [comment]
Related: PR #1174

**Comment by JulianGro** [comment]
Federation is still something that needs to get done, yes.

**Comment by strypey** [comment]
@JulianGro 
> Federation is still something that needs to get done

Have the Vircadia team looked into what the Matrix folks are doing with the Third Room project? They would definitely welcome input from potential implementers running virtual world projects:
https://thirdroom.io/

**Comment by JulianGro** [comment]
I am not a Vircadia developer anymore.
Though I assume Thirdroom uses Matrix for federation, and using Matrix as a Metaverse server/Directory server just doesn't maje much sense.
It would be a lot of extra work for no gain.
